### PR TITLE
Integrations catalog

### PR DESCRIPTION
**Pull request by Integrayshaun**

### CODE REVIEW

**Comment by winkerVSbecks** [comment]
@ShaunLloyd you'll have to update the links context https://github.com/storybookjs/components-marketing/blob/main/src/components/links-context.ts#L36 and update the components marketing package across the other sites.

**Comment by ShaunLloyd** [comment]
coverage
testrunner

**Comment by ShaunLloyd** [comment]
Actions
actions



Backgrounds
backgrounds



Controls
controls



Docs
docs



Interactions
interactions



Links
links



Source
source



Storyshots
storyshots



Viewports
viewports

File: .storybook/with-page-layout.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { PurePageLayout } from '../src/components/layout/PageLayout';
+import { dxData } from '../src/components/layout/PageLayout.stories';
+
+export function withPageLayout(story, { parameters }) {
+  const { pageLayout } = parameters;
+
+  return pageLayout ? (
```
Comment (winkerVSbecks) [inline .storybook/with-page-layout.js:8]: nice!

File: src/components/screens/RecipesDetailScreen/helpers.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import newGithubIssueUrl from 'new-github-issue-url';
+
+const REPO_URL = 'https://github.com/storybookjs/frontpage';
+
+export const generateRecipeGithubIssueLink = (recipeName) =>
```
Comment (winkerVSbecks) [inline src/components/screens/RecipesDetailScreen/helpers.js:5]: this is a very cool idea

File: src/components/screens/AddonsHomeScreen/AddonsHomeScreen.js [hunk 3]
```diff
@@ -1,95 +1,147 @@
...
-  margin-bottom: 48px;
+  margin-bottom: 3rem;
+`;
+
+const PopularRecipes = styled(RecipesList)`
+  margin-bottom: 3rem;
 `;
 
-export const AddonsHomeScreen = ({ pageContext: { popularAddons, trendingAddons } }) => {
+export const AddonsHomeScreen = ({
```
Comment (winkerVSbecks) [inline src/components/screens/AddonsHomeScreen/AddonsHomeScreen.js:101]: Should this be renamed to `IntegrationsHomeScreen`?

File: src/components/basics/FilterMenu.tsx [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+    <Icon icon="arrowdown" />
+  </Component>
+))`
+  & svg {
+    margin-left: 6px;
+    margin-right: 0;
+  }
+`;
+
+export const FilterMenu: React.VFC<FilterMenuProps> = ({
```
Comment (winkerVSbecks) [inline src/components/basics/FilterMenu.tsx:60]: Is this the same as https://github.com/storybookjs/component-catalog/blob/main/components/FilterMenu.tsx from the showcase? Does it make sense to move it into components-marketing?

File: src/components/layout/addons/AddonsAsideLayout.js [hunk 5]
```diff
@@ -25,9 +31,9 @@ export const AddonsAsideContainer = styled.div`
   }
 `;
 
-export const AddonsAside = ({ children, ...props }) => (
+export const AddonsAside = ({ children, hideLearn, ...props }) => (
```
Comment (winkerVSbecks) [inline src/components/layout/addons/AddonsAsideLayout.js:34]: Rename to `IntegrationsAside`?

File: src/components/layout/addons/AddonsLayout.js [hunk 6]
```diff
@@ -141,24 +156,39 @@ const SearchInput = styled(Input)`
...
-  margin-top: ${spacing.padding.medium}px;
   margin-bottom: ${spacing.padding.medium}px;
 `;
 
 export const SEARCH_INPUT_ID = 'addons-search';
 
 const sidebarItems = addonsToc.map((item) => ({ ...item, LinkWrapper: GatsbyLinkWrapper }));
 
-export const AddonsLayout = ({ children, data, hideSidebar, currentPath, ...props }) => {
+export const AddonsLayout = ({
```
Comment (winkerVSbecks) [inline src/components/layout/addons/AddonsLayout.js:170]: Rename for `IntegrationsLayout`

File: src/components/layout/addons/AddonsPageHeader.js [hunk 7]
```diff
@@ -32,14 +25,13 @@ const TitleWrapper = styled.div`
 `;
 
 const Header = styled.header`
-  margin-bottom: ${spacing.padding.medium}px;
+  margin-bottom: 30px;
 `;
 
-export const AddonsPageHeader = ({ title, subtitle, kicker, ...props }) => (
+export const AddonsPageHeader = ({ title, subtitle, ...props }) => (
```
Comment (winkerVSbecks) [inline src/components/layout/addons/AddonsPageHeader.js:31]: Rename?

File: src/components/layout/addons/AddonsSearchResults.js [hunk 8]
```diff
@@ -63,48 +70,83 @@ const NoAddonsFoundInner = styled.div`
...
-    {!isLoading && results.length === 0 ? (
-      <NoAddonsFound />
-    ) : (
-      <StyledAddonsList isLoading={isLoading} addonItems={results} />
-    )}
-    <AddonsAside>
-      {!isLoading && results.length > 0 && (
-        <>
-          <AddonsSubheading>Related tags</AddonsSubheading>
+export const AddonsSearchResults = ({
```
Comment (winkerVSbecks) [inline src/components/layout/addons/AddonsSearchResults.js:90]: rename?

File: src/components/screens/AddonsCategoryScreen/AddonsCategoryScreen.js [hunk 9]
```diff
@@ -1,39 +1,67 @@
...
+import { sortRecipes } from '../../../util/sort-recipes';
+import GatsbyLink from '../../basics/GatsbyLink';
+import { generateBreadcrumb } from '../../../util/generate-breadcrumb';
+
+const SortedIntegrationsList = styled(IntegrationsList)`
+  margin-bottom: 48px;
+`;
 
-export const AddonsCategoryScreen = ({ path, pageContext }) => {
+export const AddonsCategoryScreen = ({ path, location, pageContext }) => {
```
Comment (winkerVSbecks) [inline src/components/screens/AddonsCategoryScreen/AddonsCategoryScreen.js:22]: rename?

File: src/components/screens/AddonsTagScreen/AddonsTagScreen.js [hunk 10]
```diff
@@ -1,59 +1,96 @@
...
+const StyledIntegrationsList = styled(IntegrationsList)`
+  margin-bottom: 48px;
+`;
+
 const RelatedTagsList = styled(TagList)`
   margin-bottom: 48px;
 `;
 
-export const AddonsTagScreen = ({ path, pageContext, ...props }) => {
+export const AddonsTagScreen = ({ path, location, pageContext, ...props }) => {
```
Comment (winkerVSbecks) [inline src/components/screens/AddonsTagScreen/AddonsTagScreen.js:34]: rename?

File: src/hooks/use-addons-search.js [hunk 11]
```diff
@@ -47,27 +53,45 @@ export function useAddonsSearch() {
 }
 
 function searchAddons(query) {
-  return fetch('https://boring-heisenberg-43a6ed.netlify.app/', {
+  return fetch('https://deploy-preview-20--boring-heisenberg-43a6ed.netlify.app/', {
```
Comment (winkerVSbecks) [inline src/hooks/use-addons-search.js:56]: does this need updating?

**Review by winkerVSbecks — APPROVED** [review]
The renaming stuff is not a hard blocker. I would recommend making those changes to avoid confusion in the future.

File: src/components/basics/FilterMenu.tsx (hunk 4 above)
Comment (Integrayshaun) [reply src/components/basics/FilterMenu.tsx:60]: It's almost the exact same! I could certainly merge the two into FilterMenu

**Pull request merged**